database/migrations/2012_10_03_133633_create_base_tables.php

```diff
@@ -157,7 +157,7 @@ public function up(): void
                 $table->timestampTz('Submitted')->useCurrent();
                 $table->timestampTz('Edited')->nullable();
 
-                $table->index('ArticleID');
+                $table->index('ArticleID', 'comments_commentable_id_index');
```

Comment:
There actually is a `commentable_id` column in the prod db. Won't this name be confusing?

Reply:
truw, yet `ArticleID` will be dropped completely at some point as it's redundant with the polymorphic commentable relation. this index doesn't have to be renamed, really.